### ISSUE TITLE
`require is not defined` in v3.3.0

### ISSUE THREAD

**Issue by martinhecher** [rollup/rollup-plugin-commonjs#83]
We are running into a regression with v3.3.0 when creating an UMD bundle that includes the NPM module [oauthio-web](https://www.npmjs.com/package/oauthio-web).

Including the UMD bundle in a web page results in this error:

```
Uncaught ReferenceError: require is not defined
```

I created a repository to demonstrate the error:

https://github.com/Redsift/rollup-common-js-regression

Running

```
npm install
npm start
```

opens a browser window. Opening the `index.html` there shows the error in the developer console.

The code works fine with v3.0.2.

It seems that this regression could be related to the following code change:

https://github.com/rollup/rollup-plugin-commonjs/blob/c2d064d76b6ecb557cadf84e198c9b3a182300e9/src/index.js#L175

Thanks!

**Comment by MattiasBuelens** [comment]
I believe this is the same issue as #77.

OAuth.io is built with browserify, which contains `typeof require === "function" && require` in its UMD wrapper. This plugin breaks that line by replacing `typeof require` without actually providing a valid `require` function.

**Comment by Rich-Harris** [comment]
Sorry for the epic wait – fixed in 5.0.2
